icons/bit.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Bit icon</title><path d="M12 23.5C5.649 23.5.5 18.351.5 12S5.649.5 12 .5 23.5 5.649 23.5 12 18.351 23.5 12 23.5zm-1.097-10.475c0-1.239-.731-1.935-1.781-1.935-.327 0-.611.069-.826.215V9.654c0-.31-.189-.464-.576-.464s-.576.155-.576.464v4.344c0 .292 0 .576.396.8.344.198.791.292 1.273.292 1.342 0 2.09-.809 2.09-2.064zm-2.606 1.007v-1.841a.883.883 0 0 1 .542-.172c.533 0 .886.396.886 1.084 0 .679-.344 1.058-.903 1.058-.198 0-.378-.043-.525-.129zm4.761-4.078c0-.396-.284-.662-.679-.662s-.679.267-.679.662c0 .396.284.671.679.671s.679-.275.679-.671zm-.103 1.6c0-.31-.189-.464-.576-.464s-.576.155-.576.464v3.054c0 .31.189.464.576.464s.576-.155.576-.464v-3.054zm3.488 2.486c-.069 0-.129.017-.198.034-.077.017-.172.043-.267.043-.146 0-.258-.052-.318-.172-.077-.146-.077-.37-.077-.525v-1.29h.723c.31 0 .464-.155.464-.482s-.155-.482-.464-.482h-.723v-.757c0-.31-.189-.464-.576-.464s-.576.155-.576.464v.757h-.301c-.31 0-.464.155-.464.482s.155.482.464.482h.301v1.29c0 .396.017.766.189 1.075.189.344.559.594 1.178.594.335 0 .662-.077.852-.189.163-.095.206-.215.206-.335 0-.189-.095-.525-.413-.525z"/></svg>
```

Comment:
Can you make sure that the icons is _scaled to fit the viewbox, while preserving the icon's original proportions._ (as per the [Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon))?

Reply:
yes, I have checked it.

Reply:
Hmm 🤔 looking at the preview on GitHub I would disagree... Any idea what is going on here?

![image](https://user-images.githubusercontent.com/3742559/56430048-c2f27c80-62cd-11e9-8bfe-c82743aabefc.png)

Reply:
oh you are right it's was my bad.
I have fixed it :)